site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+    "./acnh-credits.png"
+    "./zla-mermaid.png"
+  >}}
+
+Additionally, also thanks to [ogniK](https://github.com/ogniK5377), `Animal Crossing: New Horizons` no longer needs the crash fix patch we were distributing in our compatibility article of the game. This was done by [improving the emulation of the Tegra X1 GPU drivers](https://github.com/yuzu-emu/yuzu/pull/4070). As a bonus, this also fixed T-posing on the later updates in `Super Smash Bros. Ultimate`!
+
+Not all games are native to the Nintendo Switch, with a portion of its library consisting of ported games from other consoles or PC. Until now, PC game ports that were originally developed for `Direct3D` and ran perfectly fine on NVN failed to run on yuzu. [Rodrigo](https://github.com/ReinUsesLisp) found a [way to translate this behavior](https://github.com/yuzu-emu/yuzu/pull/4049) to yuzu's OpenGL and Vulkan implementations, resulting in tons of western games going in game.
+
+{{< imgs
+    "./bioshock.png| Would you kindly? (BioShock Remastered)"
+  >}}
+  
+By implementing a [missing texture format](https://github.com/yuzu-emu/yuzu/pull/4005), [Rodrigo](https://github.com/ReinUsesLisp) fixed black glitches in `Xenoblade Chronicles Definitive Edition`.
+
+{{< single-title-imgs
+    "This is the Monado's power!"
+    "./xcde_before.png"
+    "./xcde_after.png"
+  >}}
+   
+{{< single-title-imgs
+    "(Xenoblade Chronicles Definitive Edition)"
+    "./xcde_before2.png"
+    "./xcde_after2.png"
+  >}}
+  
+Intel used to have a bug in their OpenGL driver causing many games using compute shaders to crash or render effects incorrectly, forcing us to disable `Compute Shaders` for any supported Intel GPU and by doing so, affecting many games. One game of note is `Super Smash Bros. Ultimate`, where compute shaders are used to render character model animations and without them, character models will be in a `T-pose`.
+
+Additionally, Intel's OpenGL `Geometry Shader` implementation, in contrast to AMD's and NVIDIA's one, is too rigorous - following the official OpenGL specification to a T due to the lack of an OpenGL extension called `GL_ARB_shader_viewport_layer_array`. This forces us to re-evaluate yuzu's implementation of it.
+
+By [re-enabling Compute Shaders](https://github.com/yuzu-emu/yuzu/pull/4025) and [changing the way yuzu handles Geometry Shaders](https://github.com/yuzu-emu/yuzu/pull/4031), [Morph](https://github.com/Morph1984) closed a big box of issues that previously affected only our Intel GPU users.
+
+
+{{< single-title-imgs
+    "Gone are the rainbows on Intel (Octopath Traveler)"
+    "./octopath-before.png"
+    "./octopath-fixed.png"
+  >}}
+  
+&nbsp;
+   
+{{< single-title-imgs
+    "No more T-posing (Super Smash Bros Ultimate)"
+    "./ssbu-tpose.png"
+    "./ssbu-fixed.png"
+  >}}
+
+[Volca](https://github.com/VolcaEM) [stubbed an input instruction](https://github.com/yuzu-emu/yuzu/pull/4032) allowing `Minecraft: Nintendo Switch Edition` to go in-game.
```

Comment:
(I'd rather not being here at all, especially since this is a stub, but) I'd add that "new Minecraft" doesn't work, only the old one, which was only available for a limited period of time
From my testing, "new" Minecraft needs many of BSD's functions implemented to work as it uses sockets a lot and stubbing those instructions isn't going to make it work unfortunately

Reply:
Do you want me to remove it?

Reply:
It's fine, but don't make it sound like a major thing plz